app/src/main/java/com/courseproject/tindar/entity/Account.java

```diff
@@ -0,0 +1,44 @@
+package com.courseproject.tindar.entity;
+
+import java.util.ArrayList;
+
+class Account {
+    int status;
```

Comment:
Pls add comment to indicate what status each integer means. Alternatively, you can have isActive boolean if active / suspended are only two status we will have.

Reply:
fixed, status is now boolean

Reply:
Can we name attribute isActiveStatus instead of status? So it is more clear.

Reply:
Done.